contracts/voting_escrow/Cargo.toml

```diff
@@ -23,21 +23,21 @@ crate-type = ["cdylib", "rlib"]
 backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cw2 = "0.8"
-cw20 = "0.8"
-cw20-base = { version = "0.8", features = ["library"] }
-cosmwasm-std = { version = "0.16" }
-cw-storage-plus = { version = "0.8.0", features = ["iterator"] }
+cw2 = "0.13.3"
```

Comment:
version 0.13.4 has already been released. Nowadays, we do not have a binding to the minor version, I recommend to specify 0.13 and for other packages too. To be updated.